jdbc/src/main/java/com/interface21/jdbc/core/JdbcExecutor.java

```diff
@@ -0,0 +1,60 @@
+package com.interface21.jdbc.core;
+
+import com.interface21.jdbc.core.exception.DataAccessException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.sql.DataSource;
+
+public class JdbcExecutor {
+
+    private final DataSource dataSource;
+
+    public JdbcExecutor(final DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    /**
+     * update 쿼리 실행용
+     */
+    public int executeUpdate(final String sql, final Object[] args) {
```

Comment:
지금은 `NamedParameterJdbc`를 사용중이라서 큰 영향은 없겠지만, 표준 SQL를 사용할 때도 고려하면 가변인자로 사용하는게 더 편리할 것 같아요 😁
```suggestion
    public int executeUpdate(final String sql, final Object... args) {
```

Reply:
감사합니다 가변 인자로 수정했습니다! 근데 가변인자로 바꾸고 저는 인자를 배열로 넘기는데 컴파일 에러가 나지 않아서 뭐지 했는데, 가변 인자에 배열도 들어간다네요!